styles/app.css

```diff
@@ -11,3 +11,86 @@ ul {
 * {
   box-sizing: border-box;
 }
+.profile {
+  height: 100vh;
+  overflow: scroll;
+  /*width: calc(2 * 100% / 3);*/
+  padding: 1rem;
+}
+
+.container {
+  /*padding: 6rem;*/
+}
+.grid {
+  /*background: blue;*/
+  overflow: auto;
+  width: 75%;
+  height: 60vh;
+  margin-top: 3.0rem;
+  margin-left: 8.5rem;
+  /*margin-right: 20rem;*/
+  /*margin-bottom:*/
+}
+.grid__item {
+  /*background: green;*/
+  border: rgba(131,131,131,.46) 0.1rem solid;
+  margin: 1rem;
+  width: calc(100% / 3 - 1rem - 1rem);
+  height: 25vh;
```

Comment:
Don't set heights on items. just let the content determine the height. You can see how this can act strangely by resizing the window height and watching the bottom border.